python/lsst/daf/butler/registry/_registry.py

```diff
@@ -554,6 +559,33 @@ def setCollectionChain(self, parent: str, children: Any) -> None:
         if children != record.children:
             record.update(self._collections, children)
 
+    def getCollectionDocumentation(self, collection: str) -> Optional[str]:
+        """Retrieve the documentation string for a collection.
+
+        Parameters
+        ----------
+        name : `str`
+            Name of the collection.
+
+        Returns
+        -------
+        docs : `str` or `None`
+            Docstring for the collection with the given name.
+        """
+        return self._collections.getDocumentation(self._collections.find(collection).key)
+
+    def setCollectionDocumentation(self, collection: str, doc: Optional[str]) -> None:
+        """Set the documentation string for a collection.
+
+        Parameters
+        ----------
+        name : `str`
+            Name of the collection.
+        docs : `str`, optional
+            Docstring for the collection with the given name.
```

Comment:
If you set doc to None here does it remove the existing docstring?  Maybe state that this value overwrites a previous value?

Reply:
It should remove the existing docstring.  I'll update the docstring and add a test to make sure it's not lying.